src/api/api.js

```diff
@@ -0,0 +1,52 @@
+import { cardData } from "../assets/data/cardData"
+
+export const deckUrl = 'https://deckofcardsapi.com/api/deck/new/'
```

Comment:
you don't need to export these two variables unless you are importing them into a different module

Reply:
Another nice catch!